app/Providers/SettingsServiceProvider.php

```diff
@@ -19,6 +20,11 @@ public function boot() {
 		if (today() > Carbon::create(date('Y'), 11, 14) && $this->app->settings->get('registration_year') == date('Y')) {
 			$year = date('Y') + 1;
 			$this->app->settings->put('registration_year', $year);
+            $users = User::where('checked_info', true)->get();
+            foreach ($users as $user){
```

Comment:
this should be done in one sql query instead of many - each loop in foreach generates new query